src/Infrastructure/Service/Setting/ProjectSettingsInitializer/ProjectFilesInitializer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\Setting\ProjectSettingsInitializer;
+
+class ProjectFilesInitializer
```

Comment:
Please, move `ProjectSettingsInitializer` to the `Infrastructure` folder. As long as almost everything in Symfony is a `service` and it is a common name, we need to get rid of it in a long run. The same for the `Question` folder.